docs/INSTALL.md

```diff
@@ -100,6 +99,14 @@ sudo apt-get update && sudo apt-get install -y \
     zlib1g-dev
 ```
 
+With Ubuntu 22.04, cmake can be installed using `apt` as well:
+
+```bash
+apt install cmake
+```
+
+For Ubuntu 20.04 though, the version that ships with Ubuntu is too old; [install it manually instead](https://cmake.org/download/).
+
```

Comment:
```suggestion
For earlier versions of Ubuntu the required cmake version can be installed using `sudo snap install cmake`
```

Reply:
Using snap works, but not if cmake is already installed via apt.

Reply:
Then add `sudo apt purge cmake`, snap is way easier than manual installation.

Reply:
See updated PR, this diff is outdated.